consultation_analyser/support_console/jinja2/support_console/consultations/show.html

```diff
@@ -17,7 +17,7 @@ <h1 class="govuk-heading-l">{{ page_title }}</h1>
 
         <p class="govuk-body">
           <a href="{{ url('consultation', kwargs={'consultation_slug': consultation.slug}) }}" class="govuk-link govuk-body govuk-link--no-visited-state">
-            View on frontend
+            View on frontend (latest results)
```

Comment:
Not necessarily, if there's no run yet?

Reply:
Now updated so "latest results" only shows if there are themes. Think it's still useful to be clear that the default themes are the latest ones.